python/llm/src/bigdl/llm/transformers/convert.py

```diff
@@ -616,6 +616,15 @@ def _optimize_post(model, lightweight_bmm=False):
         convert_forward(model,
                         module.MistralRMSNorm,
                         llama_rms_norm_forward)
+    elif model.config.model_type == "mixtral":
```

Comment:
Also check `model.config.architectures` for mistral and mixtral (to support DiscoResearch models)?

Reply:
> Also check `model.config.architectures` for mistral and mixtral (to support DiscoResearch models)?

Have added related check.